inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/MonitoringPage.html

```diff
@@ -0,0 +1,87 @@
+ - You may obtain a copy of the License at
+ -
+ - http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+-->
+<html xmlns:wicket="http://wicket.apache.org" xmlns="http://www.w3.org/1999/html" lang="">
+<head>
+  <wicket:head>
+    <style type="text/css">
+      .page-content {
+        overflow: auto;
+      }
+    </style>
+  </wicket:head>
+</head>
+<body>
+  <wicket:extend>
+    <div class="flex-content flex-v-container flex-gutter">
+      <div class="page-header">
+        <h1><wicket:container wicket:id="name"/></h1>
+      </div>
+      <form wicket:id="workload" class="flex-content flex-v-container flex-gutter flex-only-internal-gutter">
+        <div class="flex-content flex-h-container flex-gutter flex-only-internal-gutter row" style="margin-top: 0px; flex: 0.5; min-height: 500px">
+          <div class="flex-content card col-sm-3">
+            <div class="card-header">
+              <wicket:message key="settings"/>
+            </div>
+            <div class="form-group form-row" style="flex: 0.4;" wicket:enclosure="defaultDocumentsNumberTextField">
+              <label class="col-sm-9 col-form-label">
+                <wicket:message key="defaultDocumentsLabel"/>
+              </label>
+            <div class="col-sm-2">
+              <input wicket:id="defaultDocumentsNumberTextField" type="number" class="form-control"/>
+            </div>
+          </div>
+        </div>
+        <div class="flex-content card fit-child-snug col-sm-9">
+          <form wicket:id="filter" class="flex-content flex-h-container flex-gutter flex-only-internal-gutter row" style="flex: 0.4;">
+            <div class="flex-content card col-sm-7">
+              <div class="fit-child-snug">
+                <table class="table table-sm table-hover" wicket:id="dataTable" align="center" style="overflow-x:auto;"></table>
+              </div>
+            </div>
+            <div class="panel card col-sm-5 row" style="flex: 0.5;">
+              <div class="card-header">
+                <wicket:message key="filterHeader"/>
+              </div>
+              <div class="form-group form-row" style="flex: 0.4;" wicket:enclosure="filterTextfield">
+                <div class="col-sm-5" style="flex: 0.4;">
+                  <label class="col-form-label">
+                    <select wicket:id="filterDropDown"/>
+                  </label>
+                  <label class="col-form-label">
+                    <wicket:message key="unused"/>
+                  </label>
+                  <label class="col-form-label">
+                    <input wicket:id="Search" type="submit"/>
```

Comment:
For buttons you can use `class="btn btn-primary"` or `btn-secondary` to match the layout of other buttons in the application.